sklearn/neural_network/multilayer_perceptron.py

```diff
@@ -619,7 +619,9 @@ def fit(self, X, y):
 
     @property
     def partial_fit(self):
-        """Fit the model to data matrix X and target y.
+        """Fit the model to data matrix X and target y for a single iteration.
+
+        Reuses the previously fitted solution as initialization if is exists.
```

Comment:
I think using initialization is confusing here. That would be warm-start.
Maybe "do a single optimization iteration over the data, starting from the previous state"?

Reply:
That's fine with me, this new commit uses that language. (Although under-the-hood they technically work the same)

Reply:
They do not, see the glossary. ``partial_fit`` does a single iteration, ``warmstart`` solves till the optimum.

Reply:
Yes I understand. I just meant that in the context of the MLP classes, the way `partial_fit` and `warm_start` pick up from the previous stopping point is the same. But I agree using the word "initialization" is more appropriate for warm start than partial.